### PR TITLE
OSDOCS-1846: tlsSecurityProfile parameter for Kubelet as a server

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-1846

Preview: https://deploy-preview-31903--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-tls.html

### CODE REVIEW

**Comment by mburke5678** [comment]
@rphillips Can you please take a look?

File: modules/nodes-nodes-tls-configuring.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
++
+.Sample `KubeletConfig` CR for a `Custom` profile
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: set-kubelet-tls-security-profile
+spec:
+  audit:
```
Comment (rphillips) [inline modules/nodes-nodes-tls-configuring.adoc:22]: audit can be removed... Where is this coming from?

Comment (mburke5678) [reply modules/nodes-nodes-tls-configuring.adoc:22]: @rphillips It came from an earlier test case, long since edited. I will remove it here. Thanks!

**Comment by mburke5678** [comment]
@sunilcio PTAL. Thank you in advance!

**Comment by sunilcio** [comment]
@weinliu hi, could you please help review this.

**Comment by mburke5678** [comment]
Closed in favor of https://github.com/openshift/openshift-docs/pull/32826

**Pull request closed without merging**